lib/geokit/geocoders/google.rb

```diff
@@ -200,6 +200,10 @@ def self.set_precision(loc, addr)
           loc.precision = 'street'
           loc.accuracy = 7
         end
+        if addr['types'].include?('postal_code')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.